src/factories/deckCreator.js

```diff
@@ -0,0 +1,26 @@
+const CARD_SUITS = ["hearts", "spades", "diamonds", "clubs"];
+const CARD_VALUES = [
+  "A",
+  "2",
+  "3",
+  "4",
+  "5",
+  "6",
+  "7",
+  "8",
+  "9",
+  "10",
+  "J",
+  "Q",
+  "K",
+];
+
+function createDeck() {
+  return CARD_SUITS.flatMap((cardSuit) => {
+    return CARD_VALUES.map((cardValue) => {
+      return { cardSuit: cardSuit, cardValue: cardValue };
```

Comment:
Talvez seja uma questão de gosto, mas eu acho que o `card` no início da `key` é desnecessário.

Por exemplo, quando eu salvo um `User` no banco, eu tenho as colunas `name` e `age`, ao invés de `user_name`/`user_age`, porque sabemos que são atributos do `User`.

Seria o mesmo caso aqui, podemos ser mais sucintos, se a função retorna um array de cards, nós sabemos que elas tem `suit` e `value`, não precisaríamos chamar de `cartSuit`/`cardValue`.
O que acha?

```suggestion
      return { suit: suit, value: value };
```

```suggestion
      return { suit, value };
```

Reply:
Ok! :fist_oncoming: